.github/workflows/testing.yml

```diff
@@ -8,9 +8,9 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
-          - '3.0'
```

Comment:
I presume you're train of thought was to remove 3.0 because it's EOL.

What do you think about keeping 3.0 in here so we can at least get am "alarm" if something breaks in 3.0?

Reply:
Yes, you are right, it is better to leave it.